packages/types/src/chain-adapters/cosmos.ts

```diff
@@ -1,8 +1,10 @@
 export type BuildTxInput = {
   gas: string
+  memo: string
 }
 
 export type Account = {
+  account: number
```

Comment:
We are not returning an account number from unchained or the `getAccount` call. I would think this would be determined from the bip44 params passed to the function instead